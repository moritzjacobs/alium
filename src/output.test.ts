import { say, error } from "./output";

describe("output.ts", () => {
	// eslint-disable-next-line no-console
	const originalLog = console.log;
	const logMock = jest.fn(s => originalLog(s));
	global.console.log = logMock;

	it("should console.log", () => {
		say("say");
		error("error");
		expect(logMock.mock.calls).toMatchSnapshot();
	});
});
