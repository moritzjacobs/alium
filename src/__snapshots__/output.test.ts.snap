// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output.ts should console.log 1`] = `
Array [
  Array [
    "Ⓐ  say",
  ],
  Array [
    "Ⓐ  error",
  ],
]
`;
